### DISCUSSION TITLE
Linux amd64 deb for v0.90.8 missing?

### DISCUSSION THREAD

**Discussion by ups7** [TriliumNext/Notes#562] [Q&A / Support] [answered]
Hello,

The release notes for v0.90.8 state that there is a linux amd64 build now available, but can't see it in the list of assets:

![image](https://github.com/user-attachments/assets/4ec1a9bc-08cc-42d7-a72f-d516008185d3)

I'm specifically looking for the amd64 deb file. Where can I download it from?

Thanks!

**Comment by eliandoran** [comment, marked as answer]
Hello.

For the Intel/AMD x86_64 platform, the .deb file is the third one in the screenshot you have provided.
For the ARM64 platform, the .deb file is the first one.

**Reply by ups7** [reply]
Thanks for confirming. The trilium build was tagged as amd64 so I wanted to be sure.
